### DISCUSSION TITLE
Manual triggers for running the update

### DISCUSSION THREAD

**Discussion by nasiefk** [containrrr/watchtower#1085] [General] [answered]
so, we have couple of devices using watchtower and we wan't to have some of them sometimes excluded from automatic updating or trigger updating process manually through some API hook, is that possible?

**Comment by piksel** [comment, marked as answer]
Yes, see https://containrrr.dev/watchtower/http-api-mode/ (which isn't linked in the ToC for some reason).

**Reply by nasiefk** [reply]
Awesome. Thanks!
I am wondering if we somehow passed a new tag name for the image, will watchtower update to that tag?

**Reply by piksel** [reply]
No, I don't think there is any functionality like that...
